CHANGES.md

```diff
@@ -1,14 +1,126 @@
+* \[[#32](https://github.com/AcademySoftwareFoundation/Imath/pull/32)\] Tune .clang-format to match existing style
+* \[[#30](https://github.com/AcademySoftwareFoundation/Imath/pull/30)\] Changed analysis sonarcloud tests, run on pull request.
+* \[[#29](https://github.com/AcademySoftwareFoundation/Imath/pull/29)\] Added CI testing and made necessary changes to pass those tests.
+* \[[#27](https://github.com/AcademySoftwareFoundation/Imath/pull/27)\] Simplest CUDA half type conflict resolution implementation.
+* \[[#25](https://github.com/AcademySoftwareFoundation/Imath/pull/25)\] Used macros to allow compilation with C++11 and constexpr
+* \[[#24](https://github.com/AcademySoftwareFoundation/Imath/pull/24)\] b"removed pragma to disable clang's -Wself-assign-overloaded"
+* \[[#23](https://github.com/AcademySoftwareFoundation/Imath/pull/23)\] changed assert()\] to throw, which is what the original Iex macro ASSERT()\] macro did
+* \[[#21](https://github.com/AcademySoftwareFoundation/Imath/pull/21)\] First pass at adding constexpr where useful
+* \[[#20](https://github.com/AcademySoftwareFoundation/Imath/pull/20)\] Speedtest and Inversion python bindings for Arrays
+* \[[#19](https://github.com/AcademySoftwareFoundation/Imath/pull/19)\] clean up Imath repo docs
+* \[[#18](https://github.com/AcademySoftwareFoundation/Imath/pull/18)\] Transfer of PyImath history and source to Imath
+* \[[#17](https://github.com/AcademySoftwareFoundation/Imath/pull/17)\] fixed typo in README.md
+* \[[#15](https://github.com/AcademySoftwareFoundation/Imath/pull/15)\] further edits of README.md
+* \[[#14](https://github.com/AcademySoftwareFoundation/Imath/pull/14)\] First complete draft of README.md for the Imath project
+
 ## Inherited History from OpenEXR
-History dated before May 9th 2020 has been inherited from OpenEXR
-This history has been filtered using the tool [git-filter-repo](https://github.com/newren/git-filter-repo)
 
-This tool can selectively remove unneeded history based on the directories to which commits have been made. For instance, it can remove all history from a given directory using the command:\
+History dated before May 9th, 2020 has been inherited from
+https://github.com/AcademySoftwareFoundation/openexr.
+
+This history has been filtered using the tool
+[git-filter-repo](https://github.com/newren/git-filter-repo)
+
+This tool can selectively remove unneeded history based on the
+directories to which commits have been made. For instance, it can
+remove all history from a given directory using the command:
 
-git filter-repo --path old_dir1/ --path old_dir2/ --invert-paths\
+    git filter-repo --path old_dir1/ --path old_dir2/ --invert-paths
 
-Or it can keep all history from certain paths using \
+Or it can keep all history from certain paths using:
 
-git filter-repo --path remaining_dir1/ --path remaining_dir2/
+    git filter-repo --path remaining_dir1/ --path remaining_dir2/
 
```

Comment:
Is this git filter-repo tutorial helpful to readers of the release notes? Maybe it's easier to simply say "History dated before May 9th, 2020 has been inherited from the history of the OpenEXR project, omitting  commits not pertaining to the files that eventually became part of the separate Imath project."